core/src/main/java/com/scalar/db/storage/jdbc/RdbEnginePostgresql.java

```diff
@@ -184,12 +184,10 @@ public String getDataTypeForEngine(DataType scalarDbDataType) {
 
   @Override
   public String getDataTypeForKey(DataType dataType) {
-    switch (dataType) {
-      case TEXT:
-        return "VARCHAR(10485760)";
-      default:
-        return null;
+    if (dataType == DataType.TEXT) {
```

Comment:
Honestly speaking I prefer the original one in terms of extensibility 😆

Reply:
My IntelliJ complains the original one for some reason...